assets/m_avas.json

```diff
@@ -1 +1,87 @@
 {"version":0,"set_title":"Avengers Assemble"}
```

Comment:
Same problem as in #9, you need to have a `,` instead of a `}`, and in the last line the `,` need to be replaced with a `}`